SYCL/Basic/vector_byte.cpp

```diff
@@ -0,0 +1,299 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
```

Comment:
Should C++17 be explicitly selected here?

Reply:
I added option to this line